ampersand-filter-tracker.js

```diff
@@ -10,7 +10,8 @@
       filter: 'function'
     },
     session: {
-      handles: 'array'
+      handles: 'array',
+      onApply: [ 'function', false, function() { return function() {}; } ]
```

Comment:
Using `_.identity` or `_.noop` here could be nice.  Makes the intended behavior a bit more readable.  Also, why the function that returns a function?  I think this would still work and would be simpler:
```js
onApply: {type: 'function', required: false, default: _.noop}
```